src/data/token.ts

```diff
@@ -1,10 +1,18 @@
-let tokens: string[] = [];
+import {Token} from "@clarity-types/data";
 
-export function check(hash: string): boolean {
-    let output = tokens.indexOf(hash) >= 0;
-    return output;
-}
-
-export function resolve(hash: string): string[] {
-    return check(hash) ? tokens[hash] : [];
+export default function(tokens: Token[], metadata: Token[]): Token[] {
```

Comment:
What is this doing? Add some comments. Code is cryptic.

Reply:
Done